bot/code_review_bot/workflow.py

```diff
@@ -191,6 +197,41 @@ def publish(self, revision, issues, task_failures, links):
         )
         stats.add_metric("analysis.issues.publishable", nb_publishable)
 
+        if self.lando is not None:
+            logger.info(
+                "Publishing warnings to lando for {0} errors and {1} warnings".format(
+                    nb_publishable_errors, nb_publishable - nb_publishable_errors
+                ),
+                revision=revision.id,
+                diff=revision.diff["id"],
+            )
+            self.lando.del_all_warnings(revision.id, revision.diff["id"])
```

Comment:
Can you add a comment explaining that we delete all warnings here to remove the warning related to the build being in progress?